src/components/Incorporation/SummaryDefineCompany.vue

```diff
@@ -5,9 +5,13 @@
         <span>
           <v-icon color="error">mdi-information-outline</v-icon>
           <span class="error-text mx-1">This step is unfinished.</span>
-          <router-link
+          <router-link v-if="isIncorporationFiling"
             :to="{ path: `/${RouteNames.INCORPORATION_DEFINE_COMPANY}` }"
           >Return to this step to finish it</router-link>
+          <router-link v-if="isFullRestorationFiling || isLimitedRestorationFiling"
+            id="router-link"
+            :to="{ path: `/${RouteNames.RESTORATION_BUSINESS_INFORMATION}` }"
+          >Return to this step to finish it</router-link>
         </span>
```

Comment:
Review and Confirm, Return to step with validations.

![image](https://user-images.githubusercontent.com/116035339/216122513-b122f0c5-30e0-4521-bf71-15c1d714ada6.png)